Formula/hof.rb

```diff
@@ -50,10 +51,15 @@ def install
 
   test do
     ENV["HOF_TELEMETRY_DISABLED"] = "1"
-    assert_match "v#{version}", shell_output("#{bin}/hof version")
+    # upstream bug report, https://github.com/hofstadter-io/hof/issues/257
+    # assert_match "v#{version}", shell_output("#{bin}/hof version")
 
-    system bin/"hof", "mod", "init", "cue", "brew.sh/brewtest"
-    assert_predicate testpath/"cue.mods", :exist?
-    assert_equal "module: \"brew.sh/brewtest\"", (testpath/"cue.mod/module.cue").read.chomp
+    system bin/"hof", "mod", "init", "brew.sh/brewtest"
+    assert_predicate testpath/"cue.mod", :exist?
+    expected = <<~EOS
+      module: "brew.sh/brewtest"
+      cue: "#{Formula["cue"].version}"
```

Comment:
I think it's determined by [this](https://github.com/hofstadter-io/hof/blob/0d88129256d3030a5578ae4e1335a20f7ac2a2c5/cmd/hof/verinfo/verinfo.go#L18), not by what we provide.

I think it's fine to do something like

```ruby
assert_match 'module: "brew.sh/brewtest"', (testpath/"cue.mod/module.cue").read
```

instead.